### PR TITLE
Require surrender before arresting a unit

### PR DESCRIPTION
**Pull request by Glowbal**
Wednesday evening we played with the captive module between players and remote controlled AI. 

It turned out to be to easy to arrest the remote controlled players, they could not attempt to resist their arrest. This PR changes that to provide the option for having units surrender or drop their weapon first.

### CODE REVIEW

**Comment by bux578** [comment]
:+1: :)

**Comment by Glowbal** [comment]
@bux knows what I am talking about :smile:

File: addons/captives/stringtable.xml [hunk 1]
```diff
@@ -221,5 +221,11 @@
             <German>Spieler können kapitulieren, nachdem sie ihre Waffe geholstert haben</German>
             <Portuguese>Jogadores podem se render depois de guardar sua arma</Portuguese>
         </Key>
+        <Key ID="STR_ACE_Captives_ModuleSettings_requireSurrender_name">
+            <English>Require surrendering</English>
+        </Key>
```
Comment (gienkov) [inline addons/captives/stringtable.xml]: ```<Polish>Wymagaj kapitulacji</Polish>```

File: addons/captives/stringtable.xml [hunk 2]
```diff
@@ -221,5 +221,11 @@
...
+        <Key ID="STR_ACE_Captives_ModuleSettings_requireSurrender_description">
+            <English>Require Players to surrender before they can be arrested</English>
+        </Key>
```
Comment (gienkov) [inline addons/captives/stringtable.xml]: ```<Polish>Wymagaj od graczy kapitulacji zanim będzie można ich zaaresztować</Polish>```

**Comment by gienkov** [comment]
@Glowbal please
please

**Comment by PabstMirror** [comment]
:+1: I would be fine with this being default on.

Right now handcuffing is clunky.  There's no way to resist.  This way unit's have to allow themselves to be handcuffed.

**Comment by jonpas** [comment]
> I would be fine with this being default on.

As long as it still works on AI without it. :grin:

**Pull request merged**